packages/server/graphql/public/mutations/createStripeSubscription.ts

```diff
@@ -34,6 +34,12 @@ const createStripeSubscription: MutationResolvers['createStripeSubscription'] =
 
   const manager = getStripeManager()
   const {stripeId} = organization
+  if (stripeId) {
+    const existingSubscriptions = await manager.listActiveSubscriptions(stripeId)
+    if (existingSubscriptions.data.length > 0) {
+      return standardError(new Error('Organization already has a subscription'), {userId: viewerId})
+    }
+  }
```

Comment:
Not sure I fully understand what we trying to achieve here.

If we have `stripeId` saved we should have `subscriptionId` saved too.

Here we check if `stripeId` saved, subscription exist on stripe, but not saved. How is it possible?

Reply:
The `stripeId` exists if the Stripe customer exists, but a subscription could become invalid afterwards. For example, if their card expires or we cancel their subscription from the Stripe dashboard. 

That's why I thought that we shouldn't return an error if the `stripeId` exists but only if an active subscription exists.

Reply:
@nickoferrall this is correct, it is possible that `stripeId` exists but subscription is not. 

However you are checking another case. In your case `stripeId` exists and subscription exists, but data about existing subscription somehow is not stored in our database.

I wonder how is this case possible.
And if we need to handle it, we probably should record the subscription data back in our db instead of throwing an error.
Or instead of querying stripe we need to just check `organization.stripeSubscriptionId`

Reply:
I was checking Stripe instead of our db as I think it's the more important source of truth, but I'm happy to check `stripeSubscriptionId` instead. It'll help speed up the mutation.